### DISCUSSION TITLE
MobQL: A data loading library built on top of MobX

### DISCUSSION THREAD

**Discussion by Rodentman87** [mobxjs/mobx#2868] [Show and tell]
A couple weeks ago I had this idea for making a library that creates GraphQL queries, similar to what MobX did for state management. I then realized that MobX was the perfect thing to build this on top of, the reactive programming of MobX means that components can automatically re-render once the data is finished being fetched. I threw together a quick proof-of-concept, and now I've got a beta version that's likely gonna change a bit before its first stable version as this is my first public package and there's definitely a lot of changes that need to be made.

The repository is here: https://github.com/Rodentman87/mobql. Feel free to poke around and give your thoughts on the project.

**Comment by iChenLei** [comment]
cool 👍
